plugins/out_opensearch/opensearch.c

```diff
@@ -29,6 +29,7 @@
 #include <fluent-bit/flb_ra_key.h>
 #include <fluent-bit/flb_log_event_decoder.h>
 #include <msgpack.h>
+#include <fluent-bit/flb_file.h>
```

Comment:
`<fluent-bit/xxx.h> should be put in the similar headers. I mean, `<fluent-bit/flb_file.h>` should be better to put after `<fluent-bit/flb_log_event_decoder.h>`.